src/ocean/util/container/VoidBufferAsArrayOf.d

```diff
@@ -0,0 +1,220 @@
+/*******************************************************************************
```

Comment:
Not sure if `ocean.util.container` is a good location for this. Other buffery things live there, so put there by default.

Reply:
Makes sense to me.